removed invalid links, added two  correct links

RAJU2529 opened the pull request:
as per user report #9279 , so i removed two invalid links and added new links

illfated reviewed (approved):
Looks good to me. `/windows/desktop/` redirects to `/windows/win32/` already, so I approve of this change.

denisebmsft commented:
@RAJU2529, Joey (@mjcaparas) will review, but I don't think the links are invalid as they were prior to this pull request. We are told to do our links this way.

illfated commented:
Slightly odd to see issue details on docs.microsoft.com as well:
Link 1: https://docs.microsoft.com/windows/desktop/fileio/reparse-points/ => https://docs.microsoft.com/windows/win32/fileio/reparse-points/index
Link 2: https://docs.microsoft.com/windows/desktop/fileio/sparse-files/ => https://docs.microsoft.com/windows/win32/fileio/sparse-files/index
The `/index` end part of the link is what causes the 404, because if you remove the `/index` end part manually, the page shows correctly.

To me, this suggests that the issue could easily be resolved by the technicians handling the link redirections, which will make this PR redundant and the issue non-existant.

illfated commented:
I know, it is likely not much of a priority to make the document interlinking work identical on GitHub as it does on docs.microsoft.com, because I can only assume that it would need a lot of symbolic links/soft links to other repositories and non-existing directories in the MicrosoftDocs hierarchy. I also presume that managing that amount of symlinks would soon become a management issue.

RAJU2529 commented:
@denisebmsft . before creating this PR , I was verified two links, I received 404 error, so thats why I removed invalid link and added correct link, that link is given in the post. I verified that link is working and it correctly fitted.

illfated commented on windows/security/threat-protection/microsoft-defender-atp/live-response-command-examples.md:
```diff
@@ -105,8 +105,8 @@ getfile c:\Users\user\Desktop\work.txt -auto
 >
-> * [Reparse point files](/windows/desktop/fileio/reparse-points/)
-> * [Sparse files](/windows/desktop/fileio/sparse-files/)
+> * [Reparse point files](https://docs.microsoft.com/windows/win32/fileio/reparse-points)
+> * [Sparse files](https://docs.microsoft.com/windows/win32/fileio/sparse-files)
```
```suggestion
> * [Reparse point files](../../../win32/fileio/reparse-points)
> * [Sparse files](../../../win32/fileio/sparse-files)
```
I believe this is what the MS Docs team members are looking for (needs to be verified that it works as intended and expected).

RAJU2529 replied:
If possible give me full path of the link, so that after verifying I will commit change

denisebmsft replied:
@illfated and @RAJU2529, I'll defer to Joey (@mjcaparas) on what she wants to do here. The suggested revised links (../../..) are not actually what we're directed to use. We are aiming for site relative links. Agree that the links need to work properly; hence, Joey will review and decide on this pull request. We do appreciate your help with updating and maintaining the content. Thanks again for this pull request.

e0i replied:
@denisebmsft This is noted. Waiting for Joey’s input. Thanks.

mjcaparas replied:
agree with @denisebmsft - pls use the full links. What was wrong with it that it needed to be replaced with relative links? I checked both and they were working fine.

illfated replied:
Well, the docs.microsoft.com links work just fine, but they do navigate away from the github.com pages, in case someone wants to be able to navigate between the GitHub pages instead of going back to the docs.microsoft.com pages. Anyway, as long as the suggested docs.microsoft.com links are preferred, I stand down from my suggestion to change them.
